Provis.Core/Services/WorkspaceService.cs

```diff
@@ -265,5 +266,41 @@ public async Task<WorkspaceInfoDTO> GetWorkspaceInfoAsync(int workspId, string u
 
             return workspace;
         }
+
+        public async Task DeleteFromWorkspaceAsync(int workspaceId, DeleteUserDTO deleteUserDTO)
+        {
+            var user = await _userManager.FindByEmailAsync(deleteUserDTO.UserEmail);
+
+            if (user == null)
+            {
+                throw new HttpException(System.Net.HttpStatusCode.NotFound, 
+                    "User with this Id not exist");
+            }
+
+            var workspace = _workspaceRepository
+                .Query()
+                .Where(x => x.Id == workspaceId)
+                .FirstOrDefault();
```

Comment:
maby better do "_workspaceRepository.GetByKeyAsync(workspaceId)"?

Reply:
you god damn right